spawn_worker/Cargo.toml

```diff
@@ -0,0 +1,18 @@
+[package]
+edition = "2021"
+name = "spawn_worker"
+version = "0.0.1"
+
+[dependencies]
+anyhow = { version = "1.0" }
+libc = { version = "0.2" }
+memfd = { version = "0.6" }
+nix = { version = "0.26" }
+tempfile = { version = "3" }
+io-lifetimes = { version = "1.0" }
+
+[build-dependencies]
+cc_utils = { path = "../tools/cc_utils" }
+
+[dev-dependencies]
+tempfile = { version = "3.3" }
```

Comment:
Is this right? I was under the impression you wanted to expose function using this dep not just for tests

Reply:
Ah - sorry - its a leftover, from initial implementation, that only used memfd